### PR TITLE
LIME-1328 consent screen added

### PR DESCRIPTION
**Pull request by RachelRobGDS**
### What changed

New consent screen created

### Why did it change

New consent screen required by UCD as part of DL Auth Source changes.

Manual testing shows the consent screen can't be reached manually from other screens. 

DVA and DVLA screens required by UCD:
image

New DVA screen:
Screenshot 2024-11-08 at 14 06 14

New DVLA screen:
Screenshot 2024-11-08 at 14 11 23

DVA and DVLA validation error screens required by UCD:
image

New DVA validation error screen:
Screenshot 2024-11-08 at 14 10 43

New DVLA validation error screen:
Screenshot 2024-11-08 at 14 11 38

### Issue tracking

- [LIME-1328](https://govukverify.atlassian.net/browse/LIME-1328)

### Other considerations

[LIME-1328]: https://govukverify.atlassian.net/browse/LIME-1328?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/drivingLicence/controllers/check-your-details.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ const logger = require("hmpo-logger").get();
 class CheckYourDetailsController extends BaseController {
   async saveValues(req, res, next) {
     try {
+      req.sessionModel.set(
```
Comment (gds-jbentham) [inline src/app/drivingLicence/controllers/check-your-details.js:7]: Could this be set in the root controller with the other fields?

File: src/locales/cy/pages.yml [hunk 2]
```diff
@@ -25,6 +25,12 @@ check-your-details:
   issueNumber: Issue number
   postcode: Postcode
 
+consent:
```
Comment (gds-jbentham) [inline src/locales/cy/pages.yml:28]: Do we have welsh for these?

Comment (RachelRobGDS) [reply src/locales/cy/pages.yml:28]: Welsh translations are being obtain in separate [ticket 1368 ](https://govukverify.atlassian.net/browse/LIME-1368)

File: src/app/drivingLicence/controllers/check-your-details.js (hunk 1 above)
Comment (RachelRobGDS) [reply src/app/drivingLicence/controllers/check-your-details.js:7]: This is now in the root controller with the other fields

**Review by ChrisBates1 — APPROVED** [review]

**Pull request merged**